include/unifex/sequence.hpp

```diff
@@ -298,7 +298,7 @@ namespace unifex
       status status_;
       union {
         manual_lifetime<operation_t<
-            Predecessor,
+            std::remove_cvref_t<Predecessor>,
```

Comment:
This should just be `Predecessor` rather than `std::remove_cvref_t<Predecessor>` to match the type we are passing to `connect()` in the constructor.